Makefile

```diff
@@ -215,38 +213,14 @@ depend: init | $(ISTIO_OUT)
 $(ISTIO_OUT) $(ISTIO_BIN):
 	@mkdir -p $@
 
-depend.ensure: init
-
-# Target to update the Gopkg.lock with latest versions.
-# Should be run when adding any new dependency and periodically.
-depend.update: ${DEP} ; $(info $(H) ensuring dependencies are up to date...)
```

Comment:
A few lines above (where github won't let me add a comment, sigh), there's a reference to "must manually call dep.update" that probably meant "depend.update", which is now deleted in this change.

Reply:
updating that part